src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private Random rand = new Random();
+
+    private Color[] colors = Color.values();
+    private int index = rand.nextInt(colors.length);
```

Comment:
do you have every time new color?

Reply:
> do you have every time new color?

Yes
Square - Color: RED, Side: 9, Area: 81.0
Rectangle - Color: BLACK, Length: 6, Width: 1, Area: 6.0
Circle - Color: White, Radius: 10, Area: 314.1592653589793